packages/chat-widget/src/index.tsx

```diff
@@ -245,36 +279,38 @@ const saveSession = (chat: ChatHook, storeIn: StorageType) => {
   );
 };
 
-// initial eslint integration
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-export const clearSession = (storeIn: StorageType) => {
+/**
+ * Clears stored session history.
+ * @param storeIn - where to clear the session.
+ */
+export const clearSession = (storeIn: StorageType): void => {
   const storage = storeIn === "sessionStorage" ? sessionStorage : localStorage;
   storage.removeItem(storageKey);
 };
 
-export const retrieveSession = (storeIn: StorageType): SessionData | null => {
+// TODO can we stop exporting this?
+// user shouldn't have to handle managing `storeIn` config after widget is started
+const retrieveSession = (storeIn: StorageType): SessionData | null => {
```

Comment:
@peterszerzo can this library stop exporting `retrieveSession` ?

Reply:
Yes!